docs/workforce-management-framework/topics/con-logger-module.adoc

```diff
@@ -0,0 +1,11 @@
+[id='con-logger-module-{chapter}']
+= Introducing the {WFM-RC-NameShort} Logger Module
+
+The *{WFM-RC-NameLong} Logger Module* is a logging facade for link:https://nodejs.org[Node.js] based applications and it is used by all {WFM-RC-NameLong} modules.
+The *{WFM-RC-NameLong} Logger Module* is contained in the `{WFM-RC-NameLong}-core` module.
+
+Developers can use their own loggers by wrapping their implementation into the provided _logger interfaces_.
+For more information about the _logger interface_ structure, see link:{WFM-RC-CoreURL}{WFM-RC-Branch}/cloud/logger/src/Logger.ts[Logger.ts] 
```

Comment:
We only point them to the Cloud logger - should we include a client logger (or at the very least, spell out that the example is for the Cloud only)?